integration/spark/app/build.gradle

```diff
@@ -49,7 +49,7 @@ archivesBaseName='openlineage-spark-app'
 
 
 ext {
-    bigqueryVersion = '0.21.1'
+    bigqueryVersion = '0.26.0'
```

Comment:
Can we have clear wording in docs/readme whether this will now work only for Bigquery connector >=0.26.0?